R/loop_tasks.R

```diff
@@ -83,7 +83,8 @@ loop_tasks <- function(
   total_num_tasks_complete <- 0
   if (verbose){
     pb <- progress::progress_bar$new(
-      format = ":what [:bar] :percent",
+      format = ":what [:bar] :percent  ",
+      width = getOption('width')+4,
       show_after = 0, 
       clear = FALSE, total = total_num_tasks_incomplete)
```

Comment:
I haven't changed it, but I'm wondering if `total_num_tasks_incomplete` is the best choice for the progress bar total. What if it were `total_tasks = length(targets)` instead? If you're starting from a completely unbuilt remake task file, the numbers are the same, but suppose you just build 200 of 300 tasks successfully yesterday, before your computer crashed. When you restart today, would it make more sense for you to see 201/300, 202/300, etc. in the progress bar, or 1/100, 2/100, etc? The current implementation is the latter. I have a slight inclination toward the former, but I also suspect there's no universally correct answer here. What are your thoughts, @jread-usgs ?

Reply:
Yes, now that I step back from it, I think I agree. I can see a case for the current implementation, but it isn't as strong or as intuitive than changing it to your proposal.

Reply:
OK, good, I think I can pop that into this PR.